src/utils/fs.ts

```diff
@@ -1,28 +1,19 @@
-import fs from 'fs';
+import { promises as fs } from 'fs';
 import { dirname } from './path';
 
 export * from 'fs';
 
-export const readFile = (file: string): Promise<string> =>
-	new Promise<string>((fulfil, reject) =>
-		fs.readFile(file, 'utf-8', (err, contents) => (err ? reject(err) : fulfil(contents)))
-	);
+export function readFile(file: string): Promise<string> {
```

Comment:
I think we could inline this one as it only existed to do manual promise wrapping 😉

Reply:
aaah, ok. I was under the impression that it had to be in that `fs.ts` file because of the `browser` build. just a bad guess tho without having looked into the details.

in that case, I'll remove it. 👍